faq-bundle/src/Resources/contao/dca/tl_faq.php

```diff
@@ -88,7 +88,7 @@
 			(
 				'href'                => 'act=delete',
 				'icon'                => 'delete.svg',
-				'attributes'          => 'onclick="if(!confirm(\'' . $GLOBALS['TL_LANG']['MSC']['deleteConfirm'] . '\'))return false;Backend.getScrollOffset()"'
+				'attributes'          => 'onclick="if(!confirm(\'' . ($GLOBALS['TL_LANG']['MSC']['deleteConfirm'] ?? null) . '\'))return false;Backend.getScrollOffset()"'
```

Comment:
Okok. I stopped doing this annoying suggestion for `tl_faq` onwards. If we want to change this it's something best done with a regex replace anyway I assume.